goal_src/levels/common/nav-enemy.gc

```diff
@@ -768,7 +768,7 @@ nav-enemy-default-event-handler
                   )
              (>= (- (-> *display* base-frame-counter) (-> self state-time)) (-> self state-timeout))
              (nonzero? (-> self draw))
-             (logtest? (-> self draw status) (draw-status drwf03))
+            ;;  (logtest? (-> self draw status) (draw-status drwf03))
```

Comment:
can you put this in `TARGET_STARTUP_HACKS` as well, just so we don't forget about it?